client/src/components/Home.tsx

```diff
@@ -2,35 +2,45 @@ import {
     List,
     ListItem,
     Subheader,
+    FlatButton,
 } from 'material-ui';
 
-import {Course} from '../reducers/courseReducer';
+import { Course } from '../reducers/courseReducer';
 import Page from './util/Page';
 import * as React from 'react';
 
 export interface HomeProps {
     courses?: Course[];
     loading?: boolean;
     onClickCourse?: (id: number) => void;
+    onCreateCourseClick?: () => void;
+    inputText?: string;
 }
 
 export const Home = (props: HomeProps) => {
     const courses = (props.courses || []).map((c) =>
         <ListItem onClick={
-                () => props.onClickCourse ?
-                    props.onClickCourse(c.id) :
-                    undefined
-            }
+            () => props.onClickCourse ?
+                props.onClickCourse(c.id) :
+                undefined
+        }
             key={c.id}
             primaryText={c.title} />
     );
 
     let content: any;
     if (courses.length > 0) {
         content = (
-            <List style={{textAlign: 'left'}}>
+            <List style={{ textAlign: 'left' }}>
                 <Subheader>My Courses </Subheader>
                 {courses}
+                <FlatButton style={{ float: 'left', backgroundColor: 'lightgray' }} label='New Course'
```

Comment:
For consistency purposes, please change this to the same button that is used for [creating lessons](https://github.com/teamdotjs/v2/blob/master/client/src/components/Course.tsx#L61)